### PR TITLE
fix - android cipher exception

### PR DESCRIPTION
**Pull request by NoahNsimbe**
#### Summary of Changes (What does this PR do?)
- Resolves app crushing on startup due to an error in the secure storage package.

#### Is this change ready to hit production in its current state?
- [x] Yes
- [ ] No

#### Status of maturity (all need to be checked before merging):

- [x] I consider this code done
- [x] I've tested this locally
- [ ] This branch has been LGTM'ed by a reviewer

#### How should this be manually tested?
* Folllow instructions in the readme file

#### What are the relevant tickets?
- [ticket_id]()

#### Screenshots (optional)

### CODE REVIEW

File: mobile/pubspec.yaml [hunk 1]
```diff
@@ -51,7 +51,7 @@ dependencies:
   sentry_flutter: ^6.6.1
   lottie: ^1.3.0
   share_plus: ^4.0.8
-  flutter_secure_storage: ^5.0.2
+  flutter_secure_storage: ^6.0.0
```
Comment (NoahNsimbe) [inline mobile/pubspec.yaml:54]: update secure storage package

File: mobile/lib/services/secure_storage.dart [hunk 2]
```diff
@@ -9,11 +9,24 @@ class SecureStorage {
   SecureStorage._internal();
   static final SecureStorage _instance = SecureStorage._internal();
 
-  final _secureStorage = const FlutterSecureStorage();
+  final _secureStorage = const FlutterSecureStorage(
+      aOptions: AndroidOptions(
+    encryptedSharedPreferences: true,
```
Comment (NoahNsimbe) [inline mobile/lib/services/secure_storage.dart:14]: enabling use of encrypted shared preferences

File: mobile/lib/services/secure_storage.dart [hunk 3]
```diff
@@ -9,11 +9,24 @@ class SecureStorage {
...
+  ));
 
   Future<void> clearUserData() async {
-    const secureStorage = FlutterSecureStorage();
-    await secureStorage.deleteAll();
+    await _secureStorage.deleteAll();
```
Comment (Baalmart) [inline mobile/lib/services/secure_storage.dart:18]: There is this static analysis check above my comment. It is about a "trailing comma missing". Care to review this further? @NoahNsimbe

**Review by Baalmart — COMMENTED** [review]
Thanks @NoahNsimbe , this looks okay to me. Just left a small comment for you.

File: mobile/lib/services/secure_storage.dart (hunk 3 above)
Comment (NoahNsimbe) [reply mobile/lib/services/secure_storage.dart:19]: Thanks @Baalmart

**Review by BenjaminSsempala — APPROVED** [review]
LGTM

**Review by Baalmart — APPROVED** [review]
LGTM

**Pull request merged**